src/main/resources/i18n/strings.properties

```diff
@@ -124,6 +124,29 @@ unlock.error.message=Unable to unlock vault
 unlock.error.invalidMountPoint.notExisting=Mount point "%s" is not a directory, not empty or does not exist.
 unlock.error.invalidMountPoint.existing=Mount point "%s" already exists or parent folder is missing.
 unlock.error.invalidMountPoint.driveLetterOccupied=Drive Letter "%s" is already in use.
+## Hub
+### Waiting
+hub.auth.message=Waiting for authentication…
+hub.auth.description=You should automatically be redirected to the login page.
+hub.auth.loginLink=Not redirected? Click here to open it.
+### Receive Key
+hub.receive.message=Processing response…
+hub.receive.description=Cryptomator is receiving and processing the response from Hub. Please wait.
+### Register Device
+hub.register.message=Register device?
+hub.register.description=This seems to be the first access from this device to Cryptomator Hub. In order to unlock any vault, you need to choose a name and register it.
```

Comment:
```suggestion
hub.register.message=First access
hub.register.description=This seems to be the first access from this device. In order to be granted access, please choose a name.
```

Reply:
Maybe for prefilling, but I wouldn't rely on it, as in larger companies with centrally managed infrastructure, the PC names may be just some cryptic alphanumeric strings rather than "Tobis-MacBook.local"...

Reply:
I like the idea of prefilling! Implemented in https://github.com/cryptomator/cryptomator/pull/2319/commits/93500a4efd3d5d00fa675939856c4c71063ebe74